pkg/domain/podman/router.go

```diff
@@ -106,13 +111,17 @@ func (r *RouterEntityManager) QueryAllRouters() ([]qdr.Router, error) {
 		var records []qdr.Record
 		err = json.Unmarshal([]byte(rJson), &records)
 		if err != nil {
+			// routerToQuery no longer exists
+			if routerToQuery != "" {
+				continue
+			}
 			return nil, fmt.Errorf("error decoding router info from %s - %w", routerToQuery, err)
 		}
 		router.Site = qdr.GetSiteMetadata(records[0].AsString("metadata"))
 
 		// retrieving connections
 		conns, err := r.QueryConnections(routerToQuery, router.Edge)
-		if err != nil {
+		if err != nil && err.(*json.SyntaxError) == nil {
```

Comment:
Why is this extra check needed? Isn't the special case handled already in QueryConnections()?

Reply:
This is no longer needed and it has been removed now.